### PR TITLE
UCAN v0.5

### PR DESCRIPTION
**Pull request by icidasset**
Implements UCAN according to spec: https://whitepaper.fission.codes/access-control/ucan

### CODE REVIEW

File: src/apps/index.ts [hunk 1]
```diff
@@ -24,12 +25,12 @@ export async function index(): Promise<Array<App>> {
...
 
-  const jwt = await ucan.build({
+  const jwt = ucan.encode(await ucan.build({
+    attenuations: [],
     audience: await api.did(),
     issuer: await did.ucan(),
-    proof: ucan.encode(localUcan), 
-    potency: null
-  })
+    proof: ucan.encode(localUcan)
```
Comment (expede) [inline src/apps/index.ts:32]: Proofs are also an array in 0.4. Not sure if `ucan.encode` handles that yet, but I'm guessing no.

File: src/ucan/app.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export const PREFIX = "app"
+
+export type Capability
+  = "DESTROY"
+  | "REVISE"
```
Comment (expede) [inline src/ucan/app.ts:5]: `DESTROY` is now `SUPER_USER` for consistency (as of like 20 minutes ago 😅)

Comment (expede) [reply src/ucan/app.ts:5]: SUPER_USER also gives you the ability to do things like transfer ownership, which is essentially destroy+create (i.e. "move" to another user)

File: src/apps/index.ts (hunk 1 above)
Comment (icidasset) [reply src/apps/index.ts:32]: Oh thanks, I missed that.

**Comment by icidasset** [comment]
We'll use https://github.com/fission-suite/ucan instead

**Pull request closed without merging**